sdk/storage/Azure.Storage.Blobs/tests/SessionRecords/BlockBlobClientTransactionalHashingTests/OpenWriteMismatchedHashThrows(MD5).json

```diff
@@ -44,7 +44,7 @@
         "x-ms-return-client-request-id": "true",
         "x-ms-version": "2021-04-10"
       },
-      "RequestBody": [],
+      "RequestBody": null,
```

Comment:
why did these all change?

Reply:
These are due to a change in how the test proxy handles empty bodies.

Reply:
Wouldn't this affect all tests in the repo with empty bodies?

Reply:
It would only impact tests for methods that take a stream where the test explicitly passes an empty stream. There were very few such tests across the repo. This is because we have no way of distinguishing between null body and empty stream in the Test Proxy as it is out-of-process.

Reply:
In the legacy framework, empty stream would be recorded as an empty array rather than null.